core/src/main/resources/lib/hudson/executors.jelly

```diff
@@ -39,8 +39,9 @@ THE SOFTWARE.
       <j:if test="${!c.acceptingTasks}"> <st:nbsp/> (${%suspended})</j:if>
     </d:tag>
     <d:tag name="executor">
+      <j:set var="ae" value="${e.asynchronousExecution}"/>
```

Comment:
And that was where I caught my NPEs... (the issue being that `isIdle()` can have changed *after* you have captured `ae`... really need an atomic state snapshot object that executor returns in order to display its state consistently